plugins/cuda/setup.py

```diff
@@ -10,4 +11,7 @@
 build_util.bazel_build('@xla//xla/pjrt/c:pjrt_c_api_gpu_plugin.so',
                        'torch_xla_cuda_plugin/lib', ['--config=cuda'])
 
-setuptools.setup()
+setuptools.setup(
+  # TODO: Use a common version file
+  version=f'2.4.0.dev{datetime.date.today().strftime("%Y%m%d")}'
```

Comment:
lol do we need to update this for every release?

Reply:
Yeah... I do want to refactor this to use a common version file with the main build, but that's a future PR.